src/CSVReader.java

```diff
@@ -0,0 +1,76 @@
+/**
+ * Created by ken-linux2 on 14/07/16.
+ */
+import com.sun.xml.internal.ws.*;
+
+import java.io.*;
+import java.io.Closeable;
+
+
+public class CSVReader{
+    public static void main(String[] args) {
+        CSVReader RCF = new CSVReader();
+        RCF.run();
+    }
+
+    private void run() {
```

Comment:
please break this method to small private methods. We don't like to have methods that are bigger than few line. Each method should contain a logical entity, for example -private void setAccountData(...)